angular-frontend/src/app/components/data-table/table-basic.html

```diff
@@ -0,0 +1,55 @@
+<div >
+  <div class="product-dropdown">
+    <mat-form-field>
+      <mat-label>Product</mat-label>
+      <mat-select
+        [(ngModel)]="selectedValue"
+        (ngModelChange)="changeProductHandler()"
+      >
+        <mat-option value="">All</mat-option>
+        <mat-option value="LV302S">LV302</mat-option>
+        <mat-option value="MN319A">MN319A</mat-option>
+        <mat-option value="BM1249">BM1249</mat-option>
+        <mat-option value="AR3942">AR3942</mat-option>
+        <mat-option value="CH5490">CH5490</mat-option>
```

Comment:
Ideally it would have been nice to get these values from the server, but as it is you're missing the season ticket product codes